src/libraries/Microsoft.PowerFx.Core/Public/Config/FormulaValueSerializerSettings.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx
+{
+    /// <summary>
+    /// Provide settings to FormulaValues in order to serialize values in different situations.
+    /// </summary>
+    public class FormulaValueSerializerSettings
```

Comment:
>FormulaValueSerializerSettings [](http://example.com/codeflow?start=17&length=30)

Adding now to avoid breaking changes. Some potential future settings:
            // Max Depth?  
            // Overrides.   
            // .txt ... short form... "Blank()"  instead of "If(false,1)"
            // Pretty printing ... indenting?